data/src/main/scala/io/prediction/data/api/EventServer.scala

```diff
@@ -71,23 +72,26 @@ class EventServiceActor(
   val rejectionHandler = Common.rejectionHandler
 
   val jsonPath = """(.+)\.json$""".r
-  val formPath = """(.+)$""".r
+  val formPath = """(.+)\.form$""".r
```

Comment:
i had to change this to fix spray routes. There was an issue when request with path `/webhooks/segmentio.json` was matching at first by the `jsonPath` regex spray path directive but also with `formPath` regex spray path directive.